### PR TITLE
forward actions without meta.

### PR DESCRIPTION
**Pull request by coopermaruyama**
Solves issue mentioned in #45

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -28,21 +28,25 @@ function appendAction(action: Object, analytics: Object) {
...
   }
-  
+
   if (typeof options.mapper[action.type] === 'string') {
 
     let analytics = {eventType: options.mapper[action.type]};
     return handleSpec(next, appendAction(action, analytics));
   }
 
+  if (!action.meta || !action.meta.analytics) {
```
Comment (chollier) [inline src/index.js]: I wouldn't even put the if statement here

Comment (coopermaruyama) [inline src/index.js]: ah, yeah good call. updated.

**Comment by kevinhos** [comment]
It seems like `redux-segment` is broken unless this gets merged.

**Comment by chollier** [comment]
I concur

**Comment by htxiong** [comment]
when will this issue be fixed?

**Comment by gsheppard** [comment]
But like seriously. When will this get merged?

**Comment by samcorcos** [comment]
+1

**Comment by bertrandk** [comment]
This looks great. Thanks @coopermaruyama.

**Comment by bertrandk** [comment]
Sorry this took so long gents. 😓

**Comment by htxiong** [comment]
@bertrandk thanks mate. its really helpful, I am no longer need to import all your source code and patch it myself now.

**Pull request merged**